events/events-team/content/meet-and-greet.md

```diff
@@ -0,0 +1,94 @@
+# Meet And Greet Coordination
+
+The SIG Meet & Greet (M&G) is an event we hold during Kubecon in order for new contributors to find a SIG in which to start contributing.  SIG contributors occupy tables, and new and casual contributors drop by to chat.  Secondarily, it is a lunchtime event for SIG members to see each other and chat.
+
+While the M&G does not take place during the Contributor Summit, it is considered part of it.
+
+## Staffing
+
+Aside from the SIG leads, this event is run by a Meet & Greet Coordinator (MGC), a role selected on the event staff for the Contributor Summit. 
+
+## Audience
+
+The audience for the M&G consists of three groups:
+
+- SIG and WG Leads and Subproject Owners: anyone who considers themselves a "SIG/WG member" can attend and work the tables.  We particularly want SIG leads and/or Subproject owners who can guide contributors in joining their SIG and/or contributing to a project.
+- New & Casual Contributors: the visitors for the M&G will be people who are interested in contributing to Kubernetes, or have contributed lightly but not really participated in a SIG.
+- Regular Contributors: the M&G is also for regular contributors who are trying to reach out to a SIG, usually about a stuck PR, review, or mutual project.
+
+Some teams that are not SIGs or WGs may also participate, e.g. ClusterAPI or the CoCC.  There is no specific policy on this, deal with it on a case-by-case basis. Generally speaking, we want to say yes to teams that actually plan to show up as a group. In the rest of this document, "SIGs" should be considered to refer to "SIGs, WGs, and Teams".
+
+## Time and Location
+
+The M&G is generally held during lunch period on one of the main Kubecon days.  CNCF staff decides the exact schedule in order to not conflict with other lunchtime events.
+
+## Preparation & Timeline
+
+### 2-3 Months Before: Room request and SIG gear
+
+Request a large room for the M&G from the CNCF Contributor Summit Liaison.  This will be a large room, set with at least 10 round tables.
+
+The gear for the M&G consists of table sign stands for each SIG, and a bag of buttons for each.  Some of this gets used/goes missing each Kubecon, so ask CNCF staff to check that:
```

Comment:
Doesn't this also include the Kubernetes Contributor badge flags, usually? That's where I've always gotten mine, at least.

Reply:
Do you mean the patches?

Reply:
I was thinking the little banner flags that attach to the bottom of your badge. I guess most places call them badge ribbons? Like these: https://www.nametag.com/badge-ribbons/

Reply:
Hmm.  Given that the M&G often on the last day of Kubecon, I don't think we want to give those out there.  However, it would be a good idea to mention both contributor patches, and Approver pins.  Will add.